routers/repo/issue.go

```diff
@@ -1592,6 +1764,46 @@ func isLegalReviewRequest(reviewer, doer *models.User, isAdd bool, issue *models
 	return nil
 }
 
+func isLegalTeamReviewRequest(reviewer *models.Team, doer *models.User, isAdd bool, issue *models.Issue) error {
```

Comment:
hmm... Legal seems like a bad name.

isValidTeamReviewRequest would probably be better.